packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -313,6 +313,29 @@ export const S3_CREATE_DEFAULT_LOGGING_POLICY = '@aws-cdk/aws-s3:createDefaultLo
 */
 export const SNS_SUBSCRIPTIONS_SQS_DECRYPTION_POLICY = '@aws-cdk/aws-sns-subscriptions:restrictSqsDescryption';
 
+/**
+ * Enable this feature flag to get partition names as string literals in Stacks with known regions defined in
+ * their environment, such as "aws" or "aws-cn".  Previously the CloudFormation intrinsic function
+ * "Ref: AWS::Partition" was used.  For example:
+ *
+ * Principal:
+ *   AWS:
+ *     Fn::Join:
+ *       - ""
+ *       - - "arn:"
+ *         - Ref: AWS::Partition
+ *         - :iam::123456789876:root
```

Comment:
Comments are in Markdown. This needs backticks around it to be formatted in monospace.

Reply:
Thanks for your review, I'll get this fixed in the next push